src/main/scala/dpla/ingestion3/mappers/providers/TnMapping.scala

```diff
@@ -0,0 +1,265 @@
+    // <relatedItem>
+    //  <title>
+    //    <titlePart>[VALUE]
+    //  <location>
+    //    <url>[VALUE]
+    //
+    // when the type attribute DOES NOT equal "isReferencedBy" or "references" and
+    //  the "displayLabel" attribute DOES NOT equal "Project"
+    (data \\ "mods" \ "relatedItem")
+      .filterNot(node => (node \ "@type").text.equalsIgnoreCase("isReferencedBy"))
+      .filterNot(node => (node \ "@type").text.equalsIgnoreCase("references"))
+      .filterNot(node => (node \ "@displayLabel").text.equalsIgnoreCase("project"))
+      .flatMap(node =>
+        extractStrings(node \ "title" \ "titlePart") ++
+        extractStrings(node \ "location" \ "url"))
+      .map(eitherStringOrUri)
+  }
+
+  override def replacedBy(data: Document[NodeSeq]): ZeroToMany[String] =
+    // <relatedItem type="isReferencedBy">
+    //   <title>
+    //     <titlePart>[VALUE]
+    //   <location>
+    //     <url>[VALUE]
+    (data \\ "mods" \ "relatedItem")
+      .filter(node => (node \ "@type").text.equalsIgnoreCase("isReferencedBy"))
+      .flatMap(node =>
+        extractStrings(node \ "title" \ "titlePart") ++
+          extractStrings(node \ "location" \ "url"))
+
+  override def replaces(data: Document[NodeSeq]): ZeroToMany[String] =
+    // <relatedItem type="references">
+    //   <title>
+    //     <titlePart>[VALUE]
+    //   <location>
+    //     <url>[VALUE]
+    (data \\ "mods" \ "relatedItem")
+      .filter(node => (node \ "@type").text.equalsIgnoreCase("references"))
+      .flatMap(node =>
+        extractStrings(node \ "title" \ "titlePart") ++
+        extractStrings(node \ "location" \ "url"))
+
+  override def rights(data: Document[NodeSeq]): AtLeastOne[String] =
+    // <mods:accessCondition>
+    (data \\ "mods" \ "accessCondition")
+      .flatMap(node => getByAttribute(node, "type", "local rights statement")) // TODO should this filter be in place?
+      .flatMap(extractStrings)
+
+  override def subject(data: Document[NodeSeq]): Seq[SkosConcept] = {
+    // <subject>
+    //   <topic>
+    //   <name>
+    //   <titleInfo>
+    //     <title>
+    //   <genre>
+    //   <occupation>
+
+    (extractStrings(data \\ "mods" \ "subject" \ "topic") ++
+      extractStrings(data \\ "mods" \ "subject" \ "name") ++
+      extractStrings(data \\ "mods" \ "subject" \ "genre") ++
+      extractStrings(data \\ "mods" \ "subject" \ "titleInfo" \ "title") ++
+      extractStrings(data \\ "mods" \ "subject" \ "occupation"))
+      .map(nameOnlyConcept)
+  }
+
+  override def temporal(data: Document[NodeSeq]): ZeroToMany[EdmTimeSpan] =
+  // <mods:subject><mods:temporal>
+    extractStrings(data \\ "mods" \ "subject" \ "temporal")
+      .map(stringOnlyTimeSpan)
+
+  override def title(data: Document[NodeSeq]): Seq[String] =
+  // <mods:titleInfo><mods:title>
+    (data \ "metadata" \ "mods" \ "titleInfo" \ "title")
+      .filter(node => node.attributes.isEmpty)
```

Comment:
I see that you don't want to capture attributes like `type=alternative`. Just double-checking that there is no possibility that the title could have different attribute but would still qualify as a title.

Reply:
There were only 58 records missing a title and I believe all them are legitimately missing any title property.